cleanlab/count.py

```diff
@@ -410,6 +410,8 @@ def compute_confident_joint(
     true_labels_confident = true_label_guess[at_least_one_confident]
     labels_confident = labels[at_least_one_confident]
     confident_joint = confusion_matrix(true_labels_confident, labels_confident).T
+    # Guarantee at least one correctly labeled example is represented in every class
+    np.fill_diagonal(confident_joint, confident_joint.diagonal().clip(min=1))
```

Comment:
Could consider printing warning if the results are likely  unreliable if there was a 0 before.  I do not know how to judge that.

Reply:
It shouldn't make a big difference. Will leave as is for now. we can re-evaluate if this seems to cause issues down the line.